src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+	public String value;
+    protected boolean isPrivate;
+    
+    public String toString() {
+        return value;
+    }
```

Comment:
You can try to push the constructor of the subclass to the super class Contact as we discussed in the tutorial apart from these methods.

Instead of just looking at shared methods and variables, think how conceptually they are similar as well. It is useful to think in this manner when designing a super class.